rotkehlchen/types.py

```diff
@@ -81,6 +81,7 @@
 CURVE_POOL_PROTOCOL = 'curve_pool'
 PICKLE_JAR_PROTOCOL = 'pickle_jar'
 COMPOUND_PROTOCOL = 'compound'
+SPAM_PROTOCOL = 'spam'
```

Comment:
still not sure about this. Why do this? Do you want to be able to identify all assets that are inserted as spam easily in the DB?

Reply:
yes exactly for that. It might be useful to avoid operations over them or to properly identify for the users by having labels